### ISSUE TITLE
Give an option to save gpx without the images

### ISSUE THREAD

**Issue by oriori** [IsraelHikingMap/Site#771]
Feature

Sometime in the past couple of weeks the website started to include images in route points in the gpx file saved by the user.  The route point in the gpx file now looks like this, where the link href= line is 500k characters long (in this particular case):

```
<wpt lat="32.0995992285524" lon="34.9775804417118">
    <name>5</name>
    <desc>ישר לסינגל מעבר לדרך 4X4</desc>
    <link href="data:image/jpeg;base64,/9j/4Q ...
      <type>image/jpeg</type>
    </link>
    <type>star</type>
</wpt>

```
It seems many devices have problems with this type of file (so I'm reported to by people to whom I sent the file). 
I suggest changing the site back to the previous behavior where images were not included in the downloaded gpx file. At the very least, give an option to save the gpx file without the images (If you go for the give the user the option route, I would make it the default to not include images).

**Comment by oriori** [comment]
Actually, I'm not sure if this including image in the gpx behavior is a bug or a new (problematic) feature. It happens when I save this route to a file:
https://israelhiking.osm.org.il/share/wiA0UM4HG8

**Comment by HarelM** [comment]
It is caused by a new feature, which in theory will allow you to see the image offline.
I'm exhausted by trying to make the gpx file work on all the devices, especially Garmin.
Although the file might be a bit large, I assure you it is a valid file.
Have you tried contacting the manufacture of the device that doesn't work?

**Comment by oriori** [comment]
I don’t own the problematic device, I share the route with many people and
this is the response I got.
I understand your position demanding that device manufacturers adhere to
standards instead of everybody else trying to work around each device
quirks.
However, producing files that contain images seems different. It’s far from
being the de facto standard. Moreover, it can produce huge files that will
be too large for most devices.

For the time being, the solution for me will be to simply not upload images
to the routes I share.

**Comment by HarelM** [comment]
Might be solved by #765
> For the time being, the solution for me will be to simply not upload images
to the routes I share.

This is a valid workaround.

**Comment by HarelM** [comment]
I have updated the database and replaced all `data:image` with links to imgur.
When creating a share this is also done in the background and might take a few seconds to happen so sending a share to a friend and asking him to download the file will result in a small file.
As for the case of just saving a file without creating a share, I'm still not sure how to handle this, I'll think about it.
But in any case, I don't want to burden the user with the question of whether to save the images as links or as data, it needs to be handled automatically as much as possible.
